resources/common.rb

```diff
@@ -0,0 +1,27 @@
+actions :install
+default_action :install
+
+# Creates the user if they do not already exist
+attribute :create_user, 'kind_of' => [TrueClass, FalseClass], :default => false
+
+# Create the owner, path, or group if they do not exist
+attribute :path, 'kind_of' => String, :default => '/opt/app/'
```

Comment:
A default path for applications doesn't seem like a bad thing to have. An application shouldn't care where in the filesystem it lives.
A good example of a path which should be set by default and probably shouldn't be set by the user is the `virtualenv_path`. Nobody cares where that goes.
As for the `config_destination`, it seems useful to have the default be the default for all django apps. You have to go out of your way to move it.